py_ballisticcalc/settings.py

```diff
@@ -1,34 +1,39 @@
 """Global settings of the py_ballisticcalc library"""
 import logging
-
+import dataclasses
 from .unit import Unit, Distance
 
 __all__ = ('Settings',)
 
+class Metadataclass(type):
```

Comment:
Overhead! U can use `__repr__` method directly in Units class, also you need no use `dataclass` for this

```python
class Settings:  # pylint: disable=too-few-public-methods
    """Global settings class of the py_ballisticcalc library"""

    class Units:  # pylint: disable=too-few-public-methods
    class Units(metaclass=Metadataclass):  # pylint: disable=too-many-instance-attributes
        """Default units for specified measures"""
       # other code

           def __repr__(self):
               return '\n'.join(f'{name} = {value!r}'  for name, value in self.__dict__.items()
```

Reply:
I tried that and it doesn't work.  The properties are static; there's no `self`.  How can we get `__repr__` to work on a call like `str(Settings.Units)`?  (See how I use that at the top of Example.ipynb to list the default units)

Reply:
Ok, you're right